src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -333,6 +334,37 @@ function MoneyRequestView({
         ...parentReportAction?.errors,
     };
 
+    const TagList = policyTagLists.map(({name, orderWeight}, index) => {
```

Comment:
This is a variable to it should start with lowercase, right?
```suggestion
    const tagList = policyTagLists.map(({name, orderWeight}, index) => {
```